docs/conf.py

```diff
@@ -12,9 +12,9 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
```

Comment:
Why commenting and not removing the lines if they are unnecessary ?

Reply:
The code that calls `sys` and `os` are right below. I'll remove `shlex` though.